cmd/eksctl-anywhere/cmd/generateclusterconfig.go

```diff
@@ -223,24 +223,37 @@ func generateClusterConfig(clusterName string) error {
 		clusterConfigOpts = append(clusterConfigOpts, v1alpha1.WithClusterEndpoint())
 		clusterConfigOpts = append(clusterConfigOpts,
 			v1alpha1.ControlPlaneConfigCount(3),
```

Comment:
With external ETCD, can we change the default control plane count to 2 to be consistent with other EKS-A providers?

Reply:
Fixed